postcss.config.js

```diff
@@ -0,0 +1,12 @@
+const purgecss = require('@fullhuman/postcss-purgecss');
+
+module.exports = {
+  plugins: [
+    purgecss({
+      content: ['./**/*.js'],
+      safelist: ['bg-{color}', 'text-{color}'],
+      // To preserve special characters in Tailwind class names
+      defaultExtractor: content => content.match(/[\w-/:]+(?<!:)/g) || []
+    }),
+  ]
+};
```

Comment:
Thanks @yvonnetangsu ! When I attempted to scope to just src, things started to look enlarged/funky - I wasn't entirely sure why though. I tried `content: ['./src/**/*.js']` and `content: ['./src/*.js']`, let me know if I should have tried something else.

Thanks for pointing out the safelist issues, and how to test. I was attempting to replicate what I was reading in the twitter thread at https://twitter.com/adamwathan/status/1297342043400949760?lang=en, but it looks like more work is needed there.

I couldn't figure out a way to get the safelist to work with RegEx, due to the issue you mentioned about multiple things beginning with `su-bg` and `su-text` . I ended up just listing all the colors classes out, which seems to work but may not be the most elegant way. If you have a better solution, I'd love to hear!

I also tested removing the default extractor and was seeing similar issues as you mentioned.

Reply:
@jdwjdwjdw Ahh thanks for pointing me to Adam's comment on Twitter - that is very interesting! Sadly I suspect that it will only work for projects which use Tailwind's integration of PurgeCSS. In Decanter React, we aren't actually installing and configuring Tailwind in the repo, but rather just pulling in the whole CSS file from Decanter 7 and purge it down using the version of PurgeCSS that's standalone 😢  

I need to look into the other issues you mentioned some more, and let's chat about the color safelist during dev chat. I agree that the safelist thing is a bit of a struggle and there might be no real smart way to do this.